tests/framework/installer/ceph_installer.go

```diff
@@ -798,6 +802,49 @@ func (h *CephInstaller) removeClusterFinalizers(namespace, clusterName string) {
 	logger.Infof("removed finalizers from cluster %s", objectMeta.Name)
 }
 
+func (h *CephInstaller) removeMonConfigMapFinalizers(namespace, configMapName string) {
```

Comment:
This could be any configmap, right?
```suggestion
func (h *CephInstaller) removeConfigMapFinalizers(namespace, configMapName string) {
```